ansible/roles/openshift_htpasswd/templates/htpasswd.j2

```
@@ -0,0 +1,3 @@
+{% for user in oshtpasswd_users %}
```

can you add a defaults file for this variable? I think an empty list would be fine.

---

After chatting, it may be better to just have the role fail with "undefined variable" if the var is not defined correctly.